database/cyberSecurity/certificate.json

```
@@ -0,0 +1,37 @@
+[
+    {
+        "name":"CompTIA A+",
+        "description":"CompTIA A+ is entry level certificate for the person who is new in cyber security field and it is also recommended for beginners.",
+        "url":"https://www.comptia.org/certifications/a",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"Cisco CCNA",
+        "description":"Cisco provides computer network certificate to gain the knowledge of networking and its tools.",
+        "url":"https://www.cisco.com/c/en/us/training-events/training-certifications/certifications/associate/ccna.html",
+        "category":"cyberSecurity",
+        "subcategory":"certificate"
+    },
+    {
+        "name":"Certified Ethical Hacker(CEH)",
+        "description":"Certified Ethical Hacker(CEH) helps you to get good knowledge about Ethical Hacking.",
```

[Question] How does this course teach this topic? Consider using this question to write a more detailed description of this resource.
```suggestion
        "description":"Certified Ethical Hacker(CEH) helps you learn about Ethical Hacking.",
```